projects/aca-content/src/lib/ui/overrides/ay11.scss

```diff
@@ -157,13 +157,21 @@
     }
   }
 
+  .mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content {
+    outline: 2px solid var(--theme-blue-button-color);
+  }
+
   .mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content:not(.mat-datetimepicker-calendar-body-selected) {
     background-color: white !important;
-    outline: 2px solid var(--theme-blue-button-color);
+  }
+
+  .mat-datetimepicker-calendar-body-active .mat-datetimepicker-calendar-body-cell-content.mat-datetimepicker-calendar-body-selected {
+    background-color: var(--theme-grey-background-color);
```

Comment:
embedded color name should not be there

Reply:
Please see above comment

Reply:
see above, you can easly introduce a proper CSS variable and use it here, like you are doing above with the `-theme-datetimepicker-cell-background`

Reply:
Added new variables for background color on focus. New variable names are `$datetimepicker-cell-focus-background-color` &` --theme-datetimepicker-cell-focus-background`